db/schema.rb

```diff
@@ -227,6 +227,7 @@
     t.string "code_challenge"
     t.string "rails_session_id"
     t.json "verified_attributes"
+    t.datetime "verified_at"
```

Comment:
Is there a missing migration file added for this new column?

Reply:
Yeah I pushed it ...sorry